webview/src/plots/components/customPlots/CustomPlotsWrapper.tsx

```diff
@@ -0,0 +1,48 @@
+import { Section } from 'dvc/src/plots/webview/contract'
+import React, { useEffect, useState } from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import { MessageFromWebviewType } from 'dvc/src/webview/contract'
+import { CustomPlots } from './CustomPlots'
+import { changeSize } from './customPlotsSlice'
+import { PlotsContainer } from '../PlotsContainer'
+import { PlotsState } from '../../store'
+import { sendMessage } from '../../../shared/vscode'
+
+export const CustomPlotsWrapper: React.FC = () => {
+  const dispatch = useDispatch()
+  const { plotsIds, size, isCollapsed } = useSelector(
+    (state: PlotsState) => state.custom
+  )
+  const [selectedPlots, setSelectedPlots] = useState<string[]>([])
+  useEffect(() => {
+    setSelectedPlots(plotsIds)
+  }, [plotsIds, setSelectedPlots])
+
+  const handleResize = (size: number) => {
+    dispatch(changeSize(size))
+  }
```

Comment:
That makes me review my recent code at the same time. `onResize` isn't used at all anymore. We can remove it from here (and other wrappers) and remove `CommonPlotsContainerProps` as well.

Reply:
Nice catch! I'll go ahead and open a pr for removing the `onResize` in the plots code.

Reply:
#3367